bin/keeper/src/main.rs

```diff
@@ -103,6 +105,7 @@ async fn main() -> Result<(), anyhow::Error> {
                 commitment,
                 owner.clone(),
                 Some(Duration::from_secs(cli.timeout)),
+                Some(FallbackOracleConfig::Never),
```

Comment:
wouldn't it make sense for the keeper to support fallback oracles? a particular reason for Never here? (same question for the other bins and services)

Reply:
My initial thought was to start with Never and we could upgrade as necessary -- I'll amend everything to the default (Dynamic)